src_ts/jsfeatNext.ts

```diff
@@ -47,9 +47,9 @@ export default class jsfeatNext {
 
     // color conversion
     static COLOR_RGBA2GRAY = JSFEAT_CONSTANTS.COLOR_RGBA2GRAY;
-    static COLOR_RGB2GRAY = JSFEAT_CONSTANTS.COLOR_RGBA2GRAY;
-    static COLOR_BGRA2GRAY = JSFEAT_CONSTANTS.COLOR_RGBA2GRAY;
```

Comment:
This will make  COLOR_RGB2GRAY as 0 that is wrong. Fixed with the right constant.

Reply:
should be:
```diff
-JSFEAT_CONSTANTS.COLOR_RGBA2GRAY
+JSFEAT_CONSTANTS.COLOR_RGB2GRAY
```